src/types.ts

```diff
@@ -292,6 +308,8 @@ export type OutgoingAction =
   | TurnOffNamesAction
   | TurnOnSeveralTranscriptsAction
   | TurnOffSeveralTranscriptsAction
+  | TurnOnTranscriptLabelssAction
```

Comment:
labelss? my precioussss

Reply:
oopss!!!

Reply:
There were more than one! Check out the turn off action.